src/views/project/institutions/DeleteDialog.vue

```diff
@@ -31,9 +54,24 @@ async function removeInstitution(institutionIds: number[]) {
         </div>
         <div class="modal-body" v-if="institutions.length">
           Really delete institution from project:
-          <p v-for="inst in institutions" :key="inst.institutionId">
-            {{ inst.name }}
+          <p
+            v-for="institution in institutions"
+            :key="institution.institution_id"
+          >
+            {{ institution.name }}
           </p>
+          <p>
+            would you like to remove any institution from our database that is
+            not referenced by another project?
+          </p>
```

Comment:
Let's not give the user this option.

We will just remove their institutions when they are no longer referenced.